src/plugins/multifile/multifile.c

```diff
@@ -53,6 +53,7 @@ typedef enum
 	ERROR = -1,
 	NOUPDATE = 0,
 	SUCCESS = 1,
+	CACHE_HIT = 2,
```

Comment:
Isn't a single definition of this enum better? (empty seems to be yet another error, I do not think we need this.)

Reply:
> Isn't a single definition of this enum better?

You mean for all plugins in `kdbplugin.h`? I think this is actually internal state to multifile and does not necessarily match the plugin return values.

> (empty seems to be yet another error, I do not think we need this.)

The multifile resolver definitely uses the EMPTY status, to remove empty files on commit.

Reply:
Ok, then better leave it as is.